components/Footer/index.tsx

```diff
@@ -34,178 +35,172 @@ const Footer: FunctionComponent<Props> = ({ minimal }) => (
             {!minimal && (
                 <>
                     <StyledFooterRow>
-                        <StyledFooterColumn>
-                            <StyledFooterColumnHeader>About Sourcegraph</StyledFooterColumnHeader>
-                            <StyledFooterColumnList>
+                        <StyledFooterColumn />
+                        <StyledLogoFooterColumn >
+                            <a href="https://about.sourcegraph.com">
```

Comment:
Would you be onboard with the logo going to sourcegraph.com instead? I think that would keep it more up to date with changes in the dotcom